src/server.cpp

```diff
@@ -2871,12 +2871,16 @@ void Server::handleChatInterfaceEvent(ChatEvent *evt)
 }
 
 std::wstring Server::handleChat(const std::string &name, const std::wstring &wname,
-	const std::wstring &wmessage, bool check_shout_priv, RemotePlayer *player)
+	const std::wstring &wmessage_input, bool check_shout_priv, RemotePlayer *player)
```

Comment:
Using a reference here is nonsense if we're going to copy the whole string anyway.